arb_os/accounts.mini

```diff
@@ -362,6 +363,28 @@ public func account_setBaseTxFeeL1Gas(account: Account, fee: uint) -> Account {
     };
 }
 
+public view write func account_setStorage(account: Account, storage: StorageMap) -> Account {
+    return account with {
+        contractInfo: Some(
+            xif let Some(contractInfo) = account.contractInfo {
+                contractInfo with {
+                    storage: storage
+                }
+            } else {
+                let fakeCode = bytearray_new(0);
+                fakeCode = bytearray_setByte(fakeCode, 0, 0xfe);
+                struct {
+                    contractStateVersion: 1,
+                    code: fakeCode,
+                    evmJumpTable: newmap<uint, view write func()>,
+                    startCodePoint: makeTrampoline(revertPrecompile),
```

Comment:
Instead of creating a revert precompile, you can use an existing function by adding a use statement for `std::codeSegment::getDummyAcceptContractCode` then putting `startCodePoint: getDummyAccptContractCode(),` here.